plugins/auth-backend/src/providers/oidc/provider.ts

```diff
@@ -207,6 +255,24 @@ export const createOidcProvider = (
       );
       const scope = envConfig.getOptionalString('scope');
       const prompt = envConfig.getOptionalString('prompt');
+      const catalogIdentityClient = new CatalogIdentityClient({
+        catalogApi,
+        tokenIssuer,
+      });
+
+      const authHandler: AuthHandler<AuthResult> = options?.authHandler
+        ? options.authHandler
+        : async ({ userinfo, tokenset }) => ({
+            profile: makeProfileInfo({ ...userinfo } as any, tokenset.id_token), // todo some required values does not exist in oidc response
```

Comment:
what are the required values that don't exist? It could be that we'd need to hit the userinfo endpoint in some cases potentially too

Reply:
TSC error tells me:  ... is missing the following properties from type 'Profile': provider, id, displayName

Reply:
Ah you know what? Let's stick to what the provider was already doing here, so instead of using `makeProfileInfo` we'd use this as the default instead:

```ts
{
  profile: {
    displayName: userinfo.name,
    email: userinfo.email,
    picture: userinfo.picture,
  }
}
```

Reply:
changed and pushed :) thanks